examples/step-71/doc/results.dox

```diff
@@ -141,6 +141,48 @@ follow the theoretical expectations, the $H^2$-seminorm does not seem to converg
 Comparing results from $\gamma = 1$ and $\gamma = p(p+1)$, it is clear that
 $\gamma = p(p+1)$ is a better penalty.
 
+<h3>Test results on <i>Q<sub>3</sub></i> with <i>&gamma; = 1</i> </h3>
+
+<table align="center" class="doxtable">
+  <tr>
+   <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H^1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H^2}$ </th><th> Conv. rates </th>
+  </tr>
+  <tr>
+   <td>   2                  </td><td>    1.619e-02 </td><td>       </td><td>  2.282e-01  </td><td>           </td><td>  3.370 </td><td>   </td>
+  </tr>
+  <tr>
+   <td>   3                  </td><td>    4.057e-03   </td><td>  1.99  </td><td> 1.127e-01  </td><td>   1.01    </td><td> 3.179  </td><td> 0.08</td>
+  </tr>
+  <tr>
+   <td>   4                  </td><td>   1.033e-03   </td><td> 1.97   </td><td> 5.726e-02    </td><td> 0.97       </td><td>  3.22 </td><td> -0.01 </td>
+  </tr>
+  <tr>
+   <td>   5                  </td><td>   2.551e-04 </td><td>  2.01  </td><td>  2.831e-02 </td><td>   1.01    </td><td> 3.142 </td><td> 0.03   </td>
+  </tr>
+</table>
+This does not converge as theoretical expectations.
```

Comment:
For example, you could replace the whole section you introduce here by just saying something like

  Given that gamma=1 is already too small for Q_2 elements, it may not be surprising that if one repeated the
  experiment with a Q_3 element, the results are even more disappointing: One again only obtains convergence
  rates of 2, 1, zero -- i.e., no better than for the Q_2 element (although the errors are smaller in magnitude).
  Maybe surprisingly, however, one obtains more or less the expected convergence orders when using Q_4
  elements. Regardless, this uncertainty suggests that gamma=1 is at best a risky choice, and at worst an
  unreliable one and that we should choose gamma larger.